src/chains/Ethereum/index.ts

```diff
@@ -10,7 +10,7 @@ export default class Ethereum implements Chain {
     assets: [],
     txFeeInPercent: 0.1,
     module: "evm",
-    confirmLevel: 65,
+    confirmLevel: 38,
```

Comment:
This is environment dependent. Testnet requires more confirmation atm.

Reply:
ah ok. these hard-coded configs in the SDK don't accommodate environment specific changes. but the PR i have for the remote assets fetching does, and i plan to roll that out by the end of the week. so we can just stick with this for now, where the lower environment will have the mainnet configs.